src/ipld/cid_hashmap.rs

```diff
@@ -180,15 +170,14 @@ mod tests {
     }
 
     #[quickcheck]
-    fn insert_existing_key_is_some(cid_vector: Vec<(Cid, u64)>, cid: Cid, payload: u64) {
-        let (mut cid_hash_map, _) = generate_hash_maps(cid_vector);
+    fn insert_existing_key_is_some(mut cid_hash_map: CidHashMap<u64>, cid: Cid, payload: u64) {
         cid_hash_map.insert(cid, payload);
         assert!(cid_hash_map.insert(cid, payload).is_some());
     }
 
     #[quickcheck]
-    fn contains_key(cid_vector: Vec<(Cid, u64)>, cid: Cid, insert: bool) {
-        let (mut cid_hash_map, mut hash_map) = generate_hash_maps(cid_vector);
+    fn contains_key(mut cid_hash_map: CidHashMap<u64>, cid: Cid, insert: bool) {
+        let mut hash_map = HashMap::from(cid_hash_map.clone().into_iter().collect());
```

Comment:
I don't understand this code. I've you're trying to create a `std::collections::HashMap`, do `HashMap::from_iter(cid_hash_map)`